src/webgpu/api/validation/createRenderPipeline.spec.ts

```diff
@@ -201,6 +226,14 @@ g.test('sample_count_must_be_equal_to_the_one_of_every_attachment_in_the_render_
         },
       ],
     } as const;
+    const renderPipelineDescriptorWithoutDepthStencil = {
+      sampleCount: pipelineSamples,
+    } as GPURenderPipelineDescriptor;
```

Comment:
nit: use `as const` instead, if you can, as it is type safe.
(If that doesn't work, then OK to leave it like this.)